lib/medicPermissions.js

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env node
+
+/* jshint node: true */
+var path 	 = require('path');
+var fs 		 = require('fs');
+var shelljs  = require("shelljs");
+//var logger  = require('./utils').logger;
+var util     = require("./util");
+
+function MedicPermissions(appName, simulatorsFolder, tccDbPath) {
+	this.appName = appName;
+	this.simulatorsFolder = simulatorsFolder;
+	this.tccDbPath = tccDbPath;
+}
+
+MedicPermissions.prototype.updatePermissions = function(serviceList) {
```

Comment:
Though this will work, you could modify the logic as

1. Check for existence of TCC.db file. If not copy an empty file. 
2. Do the insert command. 

This will simplify the logic. Refer my latest paramedic PR for details. Also, you could reuse the same logic for finding simId instead of copying it for all simulators. 

Also, minor code changes for running the command is required. Refer latest paramedic PR for details